idris-commands.el

```diff
@@ -238,7 +238,10 @@ line."
            (lambda (_condition)
              (when (member 'warnings-tree idris-warnings-printing)
                (idris-list-compiler-notes)
-               (pop-to-buffer (idris-buffer-name :notes)))))))
+	       (if idris-stay-in-current-window-on-compiler-error
+                 (display-buffer (idris-buffer-name :notes))
+                 (pop-to-buffer (idris-buffer-name :notes)))
+	       )))))
```

Comment:
I am quite new to git, how do I squash the commits into a single commit? I guess I need to create a new PR for that right?